Header              header

geometry_msgs/Pose  pose
int8               confident
int8               matching_score
int8               loc_status